crates/rustc_codegen_spirv/src/builder/libm_intrinsics.rs

```diff
@@ -348,6 +349,8 @@ impl Builder<'_, '_> {
                 self.zombie(undef.def(self), "Scalbn not supported yet");
                 undef
             }
-        }
+             */
+            _ => return None
+        })
```

Comment:
Commenting this stuff out doesn't seem the best. I think I understand what's going on, but maybe each of these should be evaluated to see if there is a GLSL fallback and removed from the `LimCustomIntrinsic` type entirely, if not.

It would be nice if the only software operations kept would be the ones that are relatively cheap (and mostly just operate on the binary representation).